### PR TITLE
Web content

### PR DESCRIPTION
**Pull request by MilioFrankie**

### CODE REVIEW

File: src/Styles/StyledComponents.js [hunk 1]
```diff
@@ -2,14 +2,28 @@ import styled from "styled-components";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 
 // colors*********************************
-const primaryLight = "#ffffff";
-const textPrimary = "#000000";
 const primary = "#fafafa";
+const primaryLight = "#ffffff";
+// const primaryDark = "#c7c7c7";
```
Comment (matelau) [inline src/Styles/StyledComponents.js]: can you clean-up this code and remove the commented out lines?

File: src/Styles/StyledComponents.js [hunk 2]
```diff
@@ -2,14 +2,28 @@ import styled from "styled-components";
...
+const textPrimary = "#000000";
 
 //text************************************
 export const Text = styled.p`
-  color: #00000099;
+  color: #666666;
+`;
+
+export const Bold = styled.h3`
+  color: #212121;
```
Comment (matelau) [inline src/Styles/StyledComponents.js]: is this really bold? isn't it just another color?

File: src/Styles/StyledComponents.js [hunk 3]
```diff
@@ -2,14 +2,28 @@ import styled from "styled-components";
...
+`;
+
+export const TextContainer = styled.div`
+  max-width: 700px;
+  margen-left: auto;
```
Comment (matelau) [inline src/Styles/StyledComponents.js]: I don't think this is working as intended it should be `margin-left`, `margin-right`

File: src/Styles/StyledComponents.js [hunk 4]
```diff
@@ -184,6 +198,12 @@ export const Container = styled.div`
   margin-right: auto;
 `;
 
+export const SmallContainer = styled.div`
```
Comment (matelau) [inline src/Styles/StyledComponents.js]: what does this contain? Should it be named accordingly? `CodeOfConductContainer` for example

File: src/component/AboutUs.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+const primary = "#fafafa";
+const secondary = "#00bcd4";
+
+const styles = {
+  containerStyle: {
+    textAlign: "center",
+    display: "flex",
+    flexDirection: "column"
+  },
+  backgroundStyle: {
```
Comment (matelau) [inline src/component/AboutUs.js]: it is weird to me that we are using styled components and inline style. I feel like we should not have so many sources of style information.

Comment (matelau) [reply src/component/AboutUs.js]: What do you think about using a css file and importing that here instead of having these styles using classNames?

**Review by matelau — CHANGES_REQUESTED** [review]
So I have added some comments, but I what I noticed the most that I think we should clean-up is how we are adding styles to components. We have both styled components and inline style I would like us to simplify how we apply styles to where we prefer styled components and classNames to applystyles. I also think if the components need additional styling maybe they should just require a separate styled component instead of having a styled component + inline style. Let me know what you think.

File: src/StyledComponents.js [hunk 6]
```diff
@@ -213,42 +225,262 @@ export const CardGroup = styled.div`
...
+export const CareersHeaderTwo = styled.h3`
+  color: ${black};
+  margin-top: 0;
+`;
+
+export const CareersTextContainer = styled.div`
+  max-width: 700px;
+  margin-left: auto;
+  margin-right: auto;
+  alignself: center;
```
Comment (matelau) [inline src/StyledComponents.js]: shouldn't this be `align-self`?

File: src/StyledComponents.js [hunk 7]
```diff
@@ -213,42 +225,262 @@ export const CardGroup = styled.div`
...
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  color: white;
+`;
+export const DiscussionTextContainer = styled.div`
+  max-width: 700px;
+  margin-left: auto;
+  margin-right: auto;
+  alignself: center;
```
Comment (matelau) [inline src/StyledComponents.js]: `align-self`

File: src/StyledComponents.js [hunk 8]
```diff
@@ -213,42 +225,262 @@ export const CardGroup = styled.div`
...
+  display: flex;
+  flex-direction: column;
+  color: white;
+`;
+
+export const HireUsTextContainer = styled.div`
+  max-width: 700px;
+  margin-left: auto;
+  margin-right: auto;
+  alignself: center;
```
Comment (matelau) [inline src/StyledComponents.js]: `align-self`

File: src/components/Careers/index.js [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+            message on our contact page. We like résumés but we also don’t like
+            résumés. If that’s your thing, fine, if it’s not, send us a link to
+            your portfolio, blog, or whatever else you use to represent
+            yourself. We’re looking for innovators in programming, design, and
+            business. We also offer aid to bring you into our world. Inclusivity
+            doesn’t happen overnight and we feel a responsibility to create
+            doors not walls. So please don’t hesitate to drop us a message and
+            start a conversation.
+          </CareersText>
+          <button disabled>View our Code of Conduct</button>
```
Comment (matelau) [inline src/components/Careers/index.js]: can we hide this for now until we have a link?

File: src/components/Discussion/index.js [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+  DiscussionContainer,
+  DiscussionTextContainer
+} from "../../StyledComponents";
+
+const Discussion = () => (
+  <DiscussionBackground>
+    <DiscussionContainer>
+      <h3>THERE IS ALWAYS ROOM FOR DISCUSSION</h3>
+      <DiscussionTextContainer>
+        <p style={{ marginTop: "0" }}>
```
Comment (matelau) [inline src/components/Discussion/index.js]: did you mean to have this inline style? we have removed it almost everywhere else

File: src/components/HireUs/index.js [hunk 11]
```diff
@@ -0,0 +1,26 @@
...
+  HireUsContainer,
+  HireUsTextContainer
+} from "../../StyledComponents";
+
+const HireUs = () => (
+  <HireUsBackground>
+    <HireUsContainer>
+      <h3>Are you looking to hire your own team of engineers?</h3>
+      <HireUsTextContainer>
+        <p style={{ marginTop: "0" }}>
```
Comment (matelau) [inline src/components/HireUs/index.js]: more inline-style

File: src/components/NavMobileMenu/index.js [hunk 12]
```diff
@@ -1,33 +1,25 @@
...
-};
+import { MobileMenu, MenuItem, MenuList, MenuUL } from "../../StyledComponents";
 
 const NavMobileMenu = props => (
   <MobileMenu open={props.open}>
-    <ul style={styles.ulStyle}>
+    <MenuUL>
       <MenuList>
-        <MenuItem href="/">Home</MenuItem>
+        <MenuItem to="/#home">Home</MenuItem>
```
Comment (matelau) [inline src/components/NavMobileMenu/index.js]: you mentioned adding `smooth` to these

File: src/components/ProductionModel/index.js [hunk 13]
```diff
@@ -21,115 +25,101 @@ import {
...
-                  Investment is not a four-step process. You invest in our team
-                  and we invest in your future. We follow through with quality
-                  software trackmanship by unit testing, analytics, and alerts.
-                </Text>
-              </div>
-            </CardContent>
+              <Text>
+                Investment is not a four-step process. You invest in our team
+                and we invest in your future. We follow through with quality
+                software trackmanship by unit testing, analytics, and alerts.
```
Comment (matelau) [inline src/components/ProductionModel/index.js]: software `craftsmanship`

File: src/components/Title/index.js [hunk 14]
```diff
@@ -1,31 +1,18 @@
...
 
 const Title = () => (
-  <div style={styles.backgroundStyle}>
-    <Container style={{ textAlign: "center" }}>
-      <h2 style={{ color: "#F5F4F2" }}>
+  <TitleBackground id="home">
+    <TitleContainer>
+      <h2>
         We are
         <span style={{ fontSize: "150%" }}> CodePurpose </span>a one-stop dev
```
Comment (matelau) [inline src/components/Title/index.js]: inline style here

**Review by matelau — COMMENTED** [review]
several nitpicks and some word choice correction

File: src/StyledComponents.js (hunk 6 above)
Comment (MilioFrankie) [reply src/StyledComponents.js]: oops yeah, it' should be align-self

File: src/components/NavBar/index.js [hunk 15]
```diff
@@ -32,19 +32,25 @@ class NavBar extends Component {
...
             </li>
             <li>
-              <NavLinks href="/">About</NavLinks>
+              <NavLinks smooth to="/#about">
+                About
+              </NavLinks>
             </li>
             <li>
-              <NavLinks href="/">Blog</NavLinks>
+              <NavLinks to="/">Blog</NavLinks>
```
Comment (matelau) [inline src/components/NavBar/index.js:47]: can we hide the links we aren't using yet?

**Comment by mchamorro** [comment]
Need to adjust this line. We don’t have a contact page.“Next, drop us a message on our contact page.” Suggestion: “ shoot our admin an email.” Or just plain “shoot us an email.”

**Comment by mchamorro** [comment]
Man, this site is sweet. Awesome job Emilio!

**Comment by ua-hrzumaeta** [comment]
The four panels could be simplified, it's a lot of text under each. 
View our code of conduct button should be removed until the link is working.
Good job Emilio!

**Comment by hrzumaeta** [comment]
The four panels could be simplified, it's a lot of text under each. Not sure what would be better at this time however.
View our code of conduct button should be removed until the link is working.
Good job Emilio!

File: src/components/Discussion/index.js (hunk 10 above)
Comment (MilioFrankie) [reply src/components/Discussion/index.js]: replaced with a className.

File: src/components/HireUs/index.js (hunk 11 above)
Comment (MilioFrankie) [reply src/components/HireUs/index.js]: Replaced with className

File: src/components/Title/index.js (hunk 14 above)
Comment (MilioFrankie) [reply src/components/Title/index.js]: replaced with className

File: src/components/NavBar/index.js (hunk 15 above)
Comment (MilioFrankie) [reply src/components/NavBar/index.js:47]: Okay, I removed these links for now.

**Review by matelau — APPROVED** [review]

**Pull request merged**